packages/search/tests/unit/services/navi-report-search-provider-test.js

```diff
@@ -0,0 +1,72 @@
+import Service from '@ember/service';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Unit | Service | navi-report-search-provider', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function() {
+    // Load metadata needed for request fragment
+    await this.owner.lookup('service:bard-metadata').loadMetadata();
+    this.service = this.owner.lookup('service:navi-report-search-provider');
+    const store = this.owner.lookup('service:store'),
+      mockAuthor = store.createRecord('user', { id: 'navi_user' });
+    this.owner.register(
+      'service:user',
+      Service.extend({
+        getUser: () => mockAuthor
+      })
+    );
+  });
+
+  test('it exists', function(assert) {
+    const service = this.owner.lookup('service:navi-report-search-provider');
+    assert.ok(service);
+  });
+
+  test('construct full search query', function(assert) {
+    assert.deepEqual(this.service._constructSearchQuery({ title: 'Hyrule', request: 'clicks' }, 'navi_user'), {
```

Comment:
could we add text to the end explaining what the expected result is? `Report constructs the correct query for the api` or something like that